drf_integrations/migrations/0006_alter_grant_redirect_uri.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.19 on 2024-08-27 17:21
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("drf_integrations", "0005_alter_applicationinstallation_id"),
+    ]
+
+    # Django does not explicitly follow the dependency definitions under the
+    # `dependencies` variables as you might expect.
+    # It is imperative that these migrations are run BEFORE
+    # oauth2_provider.0004_auto_20200902_2022, or we get errors because the
+    # drf_integrations.Application model cannot be found
```

Comment:
this should be documented as part of the setup instructions.

Reply:
I've added a section under README.md to mention this when updating the migrations in case we come across it in the future.